finagle-netty4/src/main/scala/com/twitter/finagle/netty4/ConnectionBuilder.scala

```diff
@@ -80,6 +82,7 @@ private final class ConnectionBuilder(
         .channel(channelClass)
         .option(ChannelOption.ALLOCATOR, allocator)
         .option[JBool](ChannelOption.TCP_NODELAY, noDelay)
+        .option[JBool](UnixChannelOption.SO_REUSEPORT, reusePort)
```

Comment:
does SO_REUSEPORT make sense on the client side?

the ci logs (https://travis-ci.org/twitter/finagle/jobs/416903070) show a lot of instances of

`WARNING: Unknown channel option 'io.netty.channel.unix.UnixChannelOption#SO_REUSEPORT' for channel '[id: 0xae222ddf]'
` and I'm wondering whether it's because we're using it on the client side or whether we need to guard the setting so that we don't spam people's logs

Reply:
agree, it should be removed

Reply:
@dschobel do you want me to just ignore the presence of this setting in `ConnectionBuilder`?

Reply:
Actually, along the same lines, would it be preferable if I only add the option if `nativeEpoll.enabled` is true?

Reply:
both sound like the right call to me.